ckan/templates/package/snippets/additional_info.html

```diff
@@ -60,13 +60,22 @@
         {% if pkg_dict.metadata_modified %}
           <tr>
             <th scope="row" class="dataset-label">{{ _("Last Updated") }}</th>
-            <td class="dataset-details">{{ h.render_datetime(pkg_dict.metadata_modified, with_hours=True) }}</td>
+            <td class="dataset-details">
+                <span class="datetime" data-datetime="{{ h.render_datetime(pkg_dict.metadata_modified, date_format='%Y-%m-%dT%H:%M:%S%z') }}">
```

Comment:
data-datetime gets set to a non-UTC time when the timezone config is set, meaning we have pytz doing UTC -> tz_server and then moment.js doing tz_server -> tz_local. That *should* work but it feels fragile to me. How about putting UTC in data-datetime instead?